tests/unit/test_specfile.py

```diff
@@ -0,0 +1,52 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+import pytest
+
+from packit.specfile import Specfile
+
+spec = """
+Name: bring-me-to-the-life
+Version: 1.0
+Release: 1
+Source0: foo.bar
+License: GPLv3+
+Summary: evanescence
+%description
+-
+%changelog
+"""
+
+
+@pytest.mark.parametrize(
+    "spec_content, assertion",
+    [
+        pytest.param(
+            spec
+            + """
+%autochangelog
+""",
+            lambda sf: sf.has_autochangelog(),
```

Comment:
How about writing `True` or `False` as a parameter, rename the parameter to `has_autochangelog` and do the assertion as:

```python
assert specfile.has_autochangelog() == has_autochangelog
```

Would this be more readable?

Reply:
Yes!